src/bootstrap/dist.rs

```diff
@@ -986,17 +986,17 @@ impl Step for Src {
         // (essentially libstd and all of its path dependencies)
         let std_src_dirs = [
             "src/build_helper",
-            "src/liballoc",
-            "src/libcore",
+            "std/liballoc",
+            "std/libcore",
             "src/libpanic_abort",
             "src/libpanic_unwind",
-            "src/libstd",
+            "std/libstd",
             "src/libunwind",
-            "src/libtest",
+            "std/libtest",
             "src/libterm",
             "src/libprofiler_builtins",
             "src/stdarch",
-            "src/libproc_macro",
+            "std/libproc_macro",
             "src/tools/rustc-std-workspace-core",
             "src/tools/rustc-std-workspace-alloc",
             "src/tools/rustc-std-workspace-std",
```

Comment:
Should all crates in this list be moved? They are all part of the runtime and not the compiler.